praw/__init__.py

```diff
@@ -102,6 +102,7 @@ class Config(object):  # pylint: disable=R0903
                  'flairselector':       'api/flairselector/',
                  'flairtemplate':       'api/flairtemplate/',
                  'friend':              'api/friend/',
+                 'friend_v1':           'api/v1/me/friends/%s',
```

Comment:
Negative. Only the `{}` syntax doesn't work in python 2.6 so avoid using that, otherwise it's compatible for what we need it for. I would prefer using explicitly named parameters to make the code more readable.